lib/xclarity_client/services/update_policy_management.rb

```diff
@@ -0,0 +1,74 @@
+require 'json'
+
+module XClarityClient
+  class UpdatePolicyManagement < XClarityBase
```

Comment:
Could you follow the new structure for the `management` classes? Here you can extend `Services::XClarityService`

Reply:
thanks.. i have made respective update